Splitter/Splitter/Controllers/MyBillsViewController.swift

```diff
@@ -8,20 +8,129 @@
 
 import UIKit
 import Firebase
+import iCarousel
 
 class MyBillsViewController: UIViewController {
     
-    var titleLabel = UILabel()
-    var currentUser: SplitterUser!
+    private let addButton = AddButton(accessID: AccesID.addButton)
+    private let deleteButton = DeleteButton(accessID: AccesID.deleteButton)
+
+    private let noBillsLabel = NoBillsLabel()
+    private var titleLabel = TitleLabel()
+    private var carousel = iCarousel()
+    private var carouselDatasource = BillCarouselDatasource()
```

Comment:
these can be `let`, right?

Reply:
Yep :) e9e96da7da9fa6c56336e41714f78e8f55f067fd